test/cmdlineTests/model_checker_targets_default_all_engines/err

```diff
@@ -1,18 +1,3 @@
-Warning: CHC: Division by zero happens here.
```

Comment:
These chunks were deleted from this test and the ones below because these properties are not checked by default anymore. This is fine for the tests because we have the same tests with `targets = all` which tests the whole set.